website/docs/guides/cli.mdx

```diff
@@ -2,8 +2,6 @@
 sidebar_position: 3
 ---
 
-import CLIDownloadScript from "../../src/components/CLIDownloadScript";
```

Comment:
Might be able to rename this file to `.md` — I think it was only made `.mdx` because of this CLIDownloadScript thing